src/pages/User.vue

```diff
@@ -7,28 +7,20 @@
 
             <div class="content narrow">
 
-                <h1>Choose a trader account:</h1>
+                <form id="frmChooseTrader" v-on:submit.prevent >
 
-                <div class="form-group">
+                <h1>Choose a trader to manage</h1>
```

Comment:
It feels like the page title should not be in the form.

I think form should only contain inputs, buttons etc - not other markup.